lib/compiler-llvm/Cargo.toml

```diff
@@ -25,7 +25,8 @@ rayon = "1.5"
 loupe = "0.1"
 
 [dependencies.inkwell]
-version = "=0.1.0-beta.2"
+git = "https://github.com/wasmerio/inkwell"
+branch = "add-metadata-support"
```

Comment:
Let's make sure a version is published to creates.io (we can also create `wasmer-inkwell`) before merging